### DISCUSSION TITLE
change the style of prompt

### DISCUSSION THREAD

**Discussion by ZifengLIU143** [jspsych/jsPsych#2848] [Q&A] [answered]
Hi, 

I am conducting an audio-button response task. In this task, the content of the prompt will change according to the audio stimulus. To do this, I set a function in the prompt variable. 

`prompt: function(){return jsPsych.timelineVariable("display")}`

So, the prompt will display the text retrieved from a column name "display" in a .js file where I save the stimuli.

The program runs smoothly. When I try to change the position and the font size of the prompt, I find that it is defined as text and no style parameter allows modifying it. Is there any suggestion to fix this?

Thank you in advance,

Zifeng

**Comment by jodeleeuw** [comment, marked as answer]
Hi @ZifengLIU143,

You could add HTML around the timeline variable function, for example:

```js
prompt: function() {
  return `<div style="font-size: 40px;">${jsPsych.timelineVariable("display")}</div>`
}

**Reply by ZifengLIU143** [reply]
Hi,

Thank you for your help! I have tried adding things around the "display". It makes perfect sense to add HTML around the function!!

Have a good day!

Zifeng
